File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1079.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

n = parseInt(lines.shift())

for(i=0;i<n;i++){
    ns = (lines.shift()).split(' ')
    a = parseFloat(ns[0])
    b = parseFloat(ns[1])
    c = parseFloat(ns[2])
    console.log(((a*2+b*3+c*5)/10).toFixed(1))
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1079.py
n = int(input())

for i in range(0, n):
    v = input().split()
    a = float(v[0])
    b = float(v[1])
    c = float(v[2])
    print("{:.1f}".format((a*2+b*3+c*5)/10))